confluent_kafka/avro/serializer/MessageSerializer.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+#
+# derived from https://github.com/verisign/python-confluent-schemaregistry.git
+#
+import io
+import logging
+import struct
+import sys
+import traceback
+import avro
+import avro.io
+
+from confluent_kafka.avro import ClientError
+from . import SerializerError
+
+MAGIC_BYTE = 0
+
+HAS_FAST = False
+try:
+    from fastavro.reader import read_data
```

Comment:
@ewencp May be this can be considered as performance improvement and be part of separate PR?